src/plugins/workspace/public/components/data_source_association/association_data_source_modal.tsx

```diff
@@ -115,10 +116,9 @@ const convertConnectionToOption = ({
       : undefined,
   prepend:
     connection.connectionType === DataSourceConnectionType.DirectQueryConnection ? (
-      <>
-        <div style={{ width: 16 }} />
+      <div className="direct-query-connection-icon">
```

Comment:
```suggestion
      <div className=`${connection.connectionType === DataSourceConnectionType.DirectQueryConnection} && 'direct-query-connection-icon'`>
        <ConnectionIcon connection={connection} logos={logos} />
      <div>
```

reduces the amount of duplicate code. imo the classname would indicate to anyone who needs to modify the scss that they are modifying the icon themselves. however, they are updating the icon container.  so we should either update the classname or add the style to the icon

Reply:
Thank you, this is indeed more concise.